test/check-application

```diff
@@ -1230,7 +1230,7 @@ Yaml={name}.yaml
 
             def expectDownloadErrorForNonExistingTag(self) -> Self:
                 title = f"Danger alert:Failed to download image localhost:5000/{self.imageName}:{self.imageTag}"
-                b.wait_visible(f'h4.pf-v6-c-alert__title:contains("{title}")')
+                b.wait_in_text('h4.pf-v6-c-alert__title', title)
```

Comment:
The commit message has it backwards ("prefer wait_visible over wait_in_text") :grin: